src/facade/reply_builder_test.cc

```diff
@@ -251,14 +251,39 @@ TEST_F(RedisReplyBuilderTest, ErrorBuiltInMessage) {
   }
 }
 
+TEST_F(RedisReplyBuilderTest, ErrorReplyBuiltInMessage) {
```

Comment:
@romange This is a little "raw" but I wanted to verify that both of the execution paths are covered with `ErrorReply` without the extra nuances of printing extra error information through the `<<` operator. Moreover, it doesn't cover all sorts possible `OpStatus` values and the `e1`, `e2` are just placeholders such that I can verify that the right execution path got triggered